include/fmt/format.h

```diff
@@ -390,7 +422,8 @@ inline buffer_appender<T> reserve(buffer_appender<T> it, size_t n) {
   return it;
 }
 
-template <typename Iterator> inline Iterator& reserve(Iterator& it, size_t) {
+template <typename Iterator>
+inline constexpr Iterator& reserve(Iterator& it, size_t) {
```

Comment:
Please revert (apply clang-format).

Reply:
If you mean incorrect code-style here then it's just how clang-format works for this line, the line exceeded the column limit, that's all, I can revert this change, but it's strange... Otherwise, you could mind `constexpr` here which is necessary. Please explain what do you mean.

Reply:
Nevermind, I see you added `constexpr` which is why the formatting changed.

Reply:
Resolved 🙂